### PR TITLE
Update time-series.rst

### PR DESCRIPTION
**Pull request by NicWayand**
Thought it would be helpful to users to know that timezones are not handled here, rather than googling and finding this: https://github.com/pydata/xarray/issues/552

### CODE REVIEW

File: doc/time-series.rst [hunk 1]
```diff
@@ -60,6 +60,8 @@ native representation of dates to those that fall between the years 1678 and
 2262. When a netCDF file contains dates outside of these bounds, dates will be
 returned as arrays of ``netcdftime.datetime`` objects.
 
+Timezones are not handeled.
```
Comment (MaximilianR) [inline doc/time-series.rst]: `handled`

Comment (jhamman) [inline doc/time-series.rst]: @NicWayand - I think this is a point worth adding.  Can you expand on this a bit more though?  What are the implications for this?

**Comment by jhamman** [comment]
@NicWayand - when you have a minute, fix the typo that @MaximilianR mentioned and possibly expand the discussion a bit. Then we'll be happy to merge this in.

**Comment by NicWayand** [comment]
Numpy's datetime64 dtype currently used by xarray does not store time zone as mentioned here #552. To prevent users from making time zone errors upon dataset creation, I think the implied assumption that UTC be used, should be made more apparent in the readthedocs. Hopefully in the future it can be added to datetime64??

**Comment by shoyer** [comment]
NumPy's datetime64 is timezone naive, though indeed using UTC is generally the recommended approach. Storing timezones would need to done with another metadata layer or could be done by storing an object array of `datetime.datetime` or `pandas.Timestamp` objects.

**Pull request closed without merging**